GenderPayGap.WebUI/BusinessLogic/Services/DraftFileBusinessLogic.cs

```diff
@@ -179,12 +179,15 @@ private bool DraftExistsAndHasData(Draft draft)
         private void SetMetadata(Draft draft, long userIdRequestingAccess)
         {
             DraftReturn originalDraftReturn = GetDraftReturnFromDatabase(draft.OrganisationId, draft.SnapshotYear);
-
+            
             if (originalDraftReturn != null)
             {
+                // These 2 properties are obsolete
                 originalDraftReturn.LastWrittenByUserId = userIdRequestingAccess;
                 originalDraftReturn.LastWrittenDateTime = VirtualDateTime.Now;

```

Comment:
Are these still used anywhere? If not, we should remove these lines

Reply:
`LastWrittenByUserId` is used on line 147, 186, 223 and 344 of this file
`LastWrittenDateTime` is used on line 173, 187 and 345 of this file